h2o-hadoop/h2o-mapreduce-generic/build.gradle

```diff
@@ -16,6 +16,7 @@ compileJava {
 
 dependencies {
   compile('org.apache.hadoop:hadoop-client:' + hadoopMavenArtifactVersion)
+  compile "org.eclipse.jetty.aggregate:jetty-all-server:8.1.17.v20150415"
```

Comment:
I am getting this dependency from h2o-core, so I don't think I need to declare anything else here.